apps/playground/src/app/components/datetime/date-range-picker/date-range-picker.component.html

```diff
@@ -1,6 +1,15 @@
 <div class="date-range-demo">
   <form id="screenshot-date-range-picker" novalidate [formGroup]="reactiveForm">
-    <div class="sky-form-group">
+    <sky-input-box stacked="true">
+      <label class="sky-control-label"> First name </label>
+      <input
+        formControlName="firstName"
+        type="text"
+        spellcheck="false"
+        class="sky-form-control"
+      />
+    </sky-input-box>
+    <div>
       <sky-date-range-picker
```

Comment:
this should have a sky-margin-inline-lg on it

Reply:
ah yeah i removed it for testing, good catch 😅